platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/components/EventCreatorTests.java

```diff
@@ -471,4 +512,35 @@ void testThrottle() {
         eventCreator.createEvent(new NodeId(0));
         assertEquals(1, events.size(), "throttle should not stop event creation");
     }
+
+    @Test
+    @DisplayName("calculateNewEventCreationTime Test()")
+    void calculateNewEventCreationTimeTest() {
```

Comment:
Maybe this test (and the other `EventUtils` specific tests) should be moved to a different file?

Reply:
Moved into new test file, along with several other things that are just testing event utility methods.